packages/components/src/MultiSelect/_docs/MultiSelect.mdx

```diff
@@ -12,7 +12,11 @@ import * as MultiSelectStories from "./MultiSelect.stories"
   className="!mb-8"
 />
 
-<KaioNotification />
+<StoryInlineNotification
+  type="negative"
+  persistent
```

Comment:
I reckon bake this into the component

Reply:
I think type should probs be `cautionary`, since `negative` sentiment by default seems a little off, but defo with the persistent!